### PR TITLE
Add alternative prompts - [ENG 814]

### PR DESCRIPTION
**Pull request by varunshankar**
Add alternative prompts

### CODE REVIEW

File: data/prompts/PROMPT_README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+## Usage
+
+```
+name: LLMCommandGenerator
+prompt: path_to_prompt_file.jinja2
+```
+
+## E2E Test Results
+
+The following are the results of the e2e tests conducted for different models using the prompts:
```
Comment (OgnjenFrancuski) [inline data/prompts/PROMPT_README.md:14]: Maybe something like: 
"for different models using designated prompts"?

**Review by OgnjenFrancuski — CHANGES_REQUESTED** [review]

File: data/prompts/PROMPT_README.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+This README provides information on how to use the prompts and includes the results of the end-to-end (e2e) tests for different models.
+
+## Usage
+
+```
+name: LLMCommandGenerator
+prompt: path_to_prompt_file.jinja2
+```
+This component generates commands using a LLM based on the given prompt file and should be included in the `pipeline` section of the `config.yml` file.
```
Comment (OgnjenFrancuski) [inline data/prompts/PROMPT_README.md:11]: Maybe we could add a link to our PRO docs for LLMCommandGenerator?

File: data/prompts/PROMPT_README.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+## E2E Test Results
+
+The `e2e_tests` folder contains the test cases across different conversation categories that are used to evaluate the models.
+
+The conversations are modelled using `flows` and the `domain` file contains the definition of bot utterances, slots, and actions that are used in the test cases.
```
Comment (OgnjenFrancuski) [inline data/prompts/PROMPT_README.md:17]:
```suggestion
The conversations are modeled using `flows`. The `domain` file contains the definition of bot utterances, slots, and actions that are used in the test cases.
```

File: data/prompts/PROMPT_README.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+The following are the results of the e2e tests conducted for different models using designated prompts. 
+
+| Model   | Accuracy | Prompt file |
+|---------|----------|-------------|
+| gpt-4   | 88.09%   | default     |
+| gpt-4-1106-preview | 71.42%      | default     |
+| gpt-4-0125-preview | 67.86%      | default     |
+| gpt-3.5-turbo | 63.1%      | data/prompts/gpt_3-5_turbo_cmd_gen_prompt.jinja2     |
+| gpt-3.5-turbo-1106 | 52.38%      | data/prompts/gpt_3-5_turbo_1106_cmd_gen_prompt.jinja2     |
+| mistral-medium | 44.05%      | data/prompts/mistral_medium_cmd_gen_prompt.jinja2     |
```
Comment (OgnjenFrancuski) [inline data/prompts/PROMPT_README.md:28]: Adding a link to the actual file in the repo might be a nice touch. I think the table needs to be written in HTML. I did it with the README.md. Wdyt?

**Review by OgnjenFrancuski — COMMENTED** [review]
Great work 💯. Added a couple of small suggestions

File: data/prompts/PROMPT_README.md (hunk 4 above)
Comment (varunshankar) [reply data/prompts/PROMPT_README.md]: Yes, that's a nice idea

**Review by OgnjenFrancuski — APPROVED** [review]
Great work Varun 💯

**Pull request merged**